service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -27,8 +27,8 @@
 @Service
 public class WorkflowEngineService {
 
-  private static final String WORKFLOW_ENGINE_ACTIVATE_URL_TEMPLATE = "%s%s/workflow-engine/workflows/activate";
-  private static final String WORKFLOW_ENGINE_DEACTIVATE_URL_TEMPLATE = "%s%s/workflow-engine/workflows/deactivate";
+  private static final String WORKFLOW_ENGINE_ACTIVATE_URL_TEMPLATE = "%s/workflow-engine/workflows/activate";
+  private static final String WORKFLOW_ENGINE_DEACTIVATE_URL_TEMPLATE = "%s/workflow-engine/workflows/deactivate";
```

Comment:
Why is base path being removed?

Reply:
Because that path is not scoped to `/camunda` but at `/workflow-engine`. https://github.com/folio-org/mod-camunda/blob/master/descriptors/ModuleDescriptor-template.json#L121